README.md

```diff
@@ -3,40 +3,76 @@
 </p>
 <p align="center">
 <!-- Version badge using shields.io -->
   <a href="https://github.com/zama-ai/concrete/releases">
     <img src="https://img.shields.io/github/v/release/zama-ai/concrete?style=flat-square">
   </a>
-  <!-- Link to tutorials badge using shields.io -->
-  <a href="#license">
+  <!-- License badge -->
+  <a href="#-license">
     <img src="https://img.shields.io/badge/License-BSD--3--Clause--Clear-orange?style=flat-square">
   </a>
 <!-- Zama Bounty Program -->
   <a href="https://github.com/zama-ai/bounty-program">
     <img src="https://img.shields.io/badge/Contribute-Zama%20Bounty%20Program-yellow?style=flat-square">
   </a>
 </p>
-<hr/>
 
-**Concrete** is an open-source FHE Compiler which simplifies the use of fully homomorphic encryption (FHE).
 
-FHE is a powerful cryptographic tool, which allows computation to be performed directly on encrypted data without needing to decrypt it first. With FHE, you can build services that preserve privacy for all users. FHE is also great against data breaches as everything is done on encrypted data. Even if the server is compromised, in the end no sensitive data is leaked.
 
-Since writing FHE programs can be difficult, Concrete, based on LLVM, make this process easier for developers.
+## Table of Contents
+- **[About](#about)**
+  - [What is Concrete](#-what-is-concrete)
+  - [Main features](#-main-features)
+- **[Getting Started](#getting-started)**
+   - [Installation](#-installation)
+   - [A simple example](#-a-simple-example)
+- **[Resources](#resources)**
+   - [Concrete deep dive](#-concrete-deep-dive)
+   - [Tutorials](#-tutorials)
+   - [Documentation](#-documentation)
+- **[Working with Concrete](#working-with-concrete)**
+   - [Citations](#-citations)
+   - [Contributing](#-contributing)
+   - [License](#-license)
+- **[Support](#support)**
+<br></br>
+
+
+## About
+
+### 🟨 What is Concrete
+**Concrete** is an open-source FHE Compiler that simplifies the use of fully homomorphic encryption (FHE). Concrete framework contains a TFHE Compiler based on LLVM, making writing FHE programs an easy task for developers. 
+
+Fully Homomorphic Encryption (FHE) enables performing computations on encrypted data directly without the need to decrypt it first. FHE allows developers to build services that ensure privacy for all users. FHE is also an excellent solution against data breaches as everything is performed on encrypted data. Even if the server is compromised, no sensitive data will be leaked.
+
+Concrete is a versatile library that can be used for a variety of purposes. For instance, [Concrete ML](https://github.com/zama-ai/concrete-ml) is built on top of Concrete to simplify Machine-Learning oriented use cases.
+<br></br>
+
+
+### 🟨 Main features
 
-## Main features
+- Compiling Python functions (that may include NumPy) to their FHE equivalents, to operate on encrypted data
```

Comment:
This one seems like not a "feature" but a summary of what Concrete does. Shall we keep it here?